inst/lintr/linters.csv

```diff
@@ -71,5 +71,6 @@ undesirable_function_linter,style efficiency configurable robustness best_practi
 undesirable_operator_linter,style efficiency configurable robustness best_practices
 unneeded_concatenation_linter,style readability efficiency
 unreachable_code_linter,best_practices readability
+unused_import_linter,best_practices correctness
```

Comment:
i'm not sure `correctness` is a great tag, since there's nothing "incorrect" about having `library()` for a package you don't (or no longer) use, right?

Reply:
maybe `robustness` (because your script will run in more environments with streamlined dependencies) and/or `common_mistakes`? should we have a script-oriented counterpart to `package_development` like `scripting`?

Reply:
My reason for correctness is the dev probably intended to use the package so maybe forgot a piece of code that acutally uses the package. But open to changing this to `common_mistakes`. Robustness is intended for "Linters highlighting code robustness issues, such as possibly wrong edge case behaviour.", not really fitting here IMO.

Regarding scripting: Might be useful. Let's do this in a separate PR.

Reply:
for `robustness`, I had in mind "this script will run in more places if it excludes unneeded `library()` calls", but agree it's a stretch.

I think `common_mistakes` is better -- the other situation is when the package _was_ used in the past, but that code's now deleted / migrated elsewhere, so this script can drop the requirement. I think `common_mistakes` covers the two cases better than `correctness`.